shipping/uc_usps/config/uc_usps.settings.json

```
@@ -11,5 +11,73 @@
     "uc_usps_weight_markup_type": "percentage",
     "uc_usps_weight_markup": 0,
     "uc_usps_markup": "",
-    "uc_usps_markup_type": ""
-}
\ No newline at end of file
+    "uc_usps_markup_type": "",
+    "uc_usps_authorization_method": "newapi",
+    "uc_usps_consumer_key": "",
+    "uc_usps_consumer_secret": "",
+    "uc_usps_connection_address": "https://apis-tem.usps.com",
+    "uc_usps_price_type": "COMMERCIAL",
+    "uc_usps_newapi_dom_env_services": {
+        "FCLM": "FCLM",
+        "FCP": "FCP",
+        "FCF": "FCF",
+        "PMFRE": "PMFRE",
+        "PMPFRE": "PMPFRE",
+        "PMLFRE": "PMLFRE",
+        "PMMS": "PMMS",
+        "PMEFRE": "PMEFRE",
+        "PMEPFRE": "PMEPFRE",
+        "PMELFRE": "PMELFRE",
+        "PMELFREHD": "PMELFREHD",
+        "PMEMS": "PMEMS",
+        "UGAMS": "UGAMS"
+    },
+    "uc_usps_newapi_dom_parcel_services": {
+        "FCLM": "FCLM",
+        "FCF": "FCF",
+        "PMMSFRB": "PMMSFRB",
+        "PMMMFRB": "PMMMFRB",
+        "PMMLFRB": "PMMLFRB",
+        "PMMLFRBA": "PMMLFRBA",
+        "PMMS": "PMMS",
+        "PMMCNPT1": "PMMCNPT1",
+        "PMMCSPT1": "PMMCSPT1",
+        "PMEMS": "PMEMS",
+        "UGAMS": "UGAMS",
+        "UGAMCNPT1": "UGAMCNPT1",
+        "UGAMCSPT1": "UGAMCSPT1",
+        "BPMMP": "BPMMP",
+        "BPMNP": "BPMNP",
+        "LMNB": "LMNB",
+        "LMNS": "LMNS",
```

```suggestion
        "LMNS": "LMNS",
        "LMMS": "LMMS",
```

I had added a suggestion for LMMS and MMMS to `uc_usps_newapi_dom_parcel_all_services()` but forgot to add it to the default settings.